solo-chains/runtime/dancelight/src/lib.rs

```diff
@@ -1335,17 +1336,39 @@ impl pallet_external_validators::Config for Runtime {
     type UnixTime = Timestamp;
     type SessionsPerEra = SessionsPerEra;
     type OnEraStart = (ExternalValidatorSlashes, ExternalValidatorsRewards);
-    type OnEraEnd = ();
+    // reminder to add on_initialize weight
+    type OnEraEnd = ExternalValidatorsRewards;
```

Comment:
How do we account for this? OnEraEnd is run in on_initialize but we have no way to return the weight, and the pallet doesn't know when does the era end so we can't just add the weight to on_initialize.

Should we return the weight from the OnEraEnd method? Or just benchmark it and call register_extra_weight_unchecked?

Reply:
I was just thinking about that. In my opinion I would return the weight from the `OnEraEnd` method, and I guess in that case we would also need to modify the `on_initialize` method where this is called.

Reply:
After analyzing it a bit more, I think it's better to go for the benchmarking option. I'll be working on that.

Reply:
Added the register_extra_weight_unchecked function with the proper benchmarked weight in 74537a6c8ae42cd99f4ae5a0e6f2c50ff971d8a0.

Reply:
_fixed_